### DISCUSSION TITLE
A message for all the devs.

### DISCUSSION THREAD

**Discussion by LNLenost** [tfdsoft/famidash#274] [General]
(Before starting, I already wrote to kando on yt but wanted to say this to the rest of your team)

Hello everyone, it's me. Your worst nightmare is back :)

1.0 released, and... wow. I am geniunely amazed of how good the game turned out, and the fact that levels are identical to the real ones.

I wanted to make my congrats to all of the team for this masterpiece, and I promise I will buy the cartdrige version as soon as it releases. GG!

That's all from me.
/LNLenost

**Comment by glacierjl** [comment]
Welcome back and thanks for the cake- I mean feedback. We worked our asses off for this, but there's yet more to come. ~~Technically I didn't really do anything but that's not the point~~
Anyways, feel free to follow along. And behave. ~~or else~~

**Reply by LNLenost** [reply]
🤣
I will <3

**Comment by kandowontu** [comment]
Welcome back! Fresh start, enjoy the game :)
